src/libraries/Common/src/Interop/Windows/WinSock/Interop.WSARecvFrom.cs

```diff
@@ -53,7 +53,7 @@ internal static partial class Winsock
             IntPtr completionRoutine)
         {
             Debug.Assert(buffers != null && buffers.Length > 0);
-            fixed (WSABuffer* buffersPtr = &buffers[0])
+            fixed (WSABuffer* buffersPtr = buffers)
```

Comment:
Similarly, wondering if this should use MemoryMarshal.GetReference?  We've generally preferred to use it if the method being called doesn't require null for a zero-length span, though the tiny savings probably isn't measurable.

Reply:
Changed to GetReference